docs/reference/mapping/fields/routing-field.asciidoc

```diff
@@ -109,3 +109,28 @@ documents with the same `_id` might end up on different shards if indexed with
 different `_routing` values.
 
 It is up to the user to ensure that IDs are unique across the index.
+
+==== Routing to an index partition
```

Comment:
Add an ID for linking

    [[routing-index-partition]]
    ==== Routing to an index partition